### ISSUE TITLE
Examples: The thumbnails toggle causes a visual error when searching

### ISSUE THREAD

**Issue by gsimone** [mrdoob/three.js#20208]
This happens when the results are filtered and the first category (webgl) is hidden.

<img src="https://user-images.githubusercontent.com/1862172/91550604-1268a400-e929-11ea-8a22-6c382f7fa910.png" width="300px" />

This could be solved with this CSS:
```css
.card:not(.hidden) ~ h2 {
    margin-top: 32px;
    border-top: var(--border-style);
    padding-top: 12px;
}
```

The selector matches any h2 that is a preceded by a non-hidden card:
<img src="https://user-images.githubusercontent.com/1862172/91551495-85bee580-e92a-11ea-80ef-c98603368ec0.png" width="300px" />

This way the margin, padding and borders are only applied when necessary.

**Comment by Mugen87** [comment]
Related: #19943

**Comment by Mugen87** [comment]
Does your fix depend on #20207? If not, can you please file a PR as a replacement of #19943?

**Comment by gsimone** [comment]
No, I just noticed while working on that, I'll make a separate PR 👍

**Comment by Mugen87** [comment]
Thanks!
